src/test/java/com/rultor/agents/daemons/StartsDaemonITCase.java

```diff
@@ -115,27 +120,33 @@ public void startsDaemon() throws Exception {
     }
 
     /**
-     * StartsDaemon can deprecate default image.
+     * StartsDaemon can deprecate default image (except one case, when
+     * repo is is the actual Rultor repo: https://github.com/yegor256/rultor).
      * @throws IOException In case of error
-     * @todo #1018:30min This test should not check for start and end
-     *  deprecation messages if when the repo is the actual Rultor repo:
-     *  https://github.com/yegor256/rultor
      */
     @Test
-    @Ignore
     public void deprecatesDefaultImage() throws IOException {
         final Talk talk = this.talk();
         final XML xml = talk.read();
-        for (final String path
-            : xml.xpath("/p/entry[@key='merge']/entry[@key='script']")
-             ) {
-            if ("yegor256/rultor".equals(path)) {
-                final String dir = talk.read()
-                    .xpath("/talk/daemon/dir/text()").get(0);
-                MatcherAssert.assertThat(
-                    dir,
-                    StringStartsWith.startsWith("#### Deprecation Notice ####")
-                );
+        final List<String> repos = xml.xpath("/wire/github-repo/text()");
+        if (
```

Comment:
@xupyprmv just for my peace of mind :) This looks just fine to me, but I'm slightly fearful someone might break `this.talk()` inadvertently ...
Can we just adjust this in a trivial manner please ? :)

* move  `final String dir` in front of the first `if` ( l.131 currently)
* Move this section at the end of the function
```java
                    MatcherAssert.assertThat(
                         dir,
                         StringStartsWith.startsWith(
                             "#### Deprecation Notice ####"
                         )
                     );
```

This way this will always and forever work and also is easier to read imo :)